A3A/addons/core/Params.hpp

```diff
@@ -331,6 +330,35 @@ class Params
         texts[] = {$STR_antistasi_dialogs_generic_button_no_text,$STR_antistasi_dialogs_generic_button_yes_text};
         default = 1;
     };
+    class RRTurretMagazines
+    {
+        title = $STR_params_ReloadRepackTurretMagazines;
+        values[] = {0,1};
+        texts[] = {$STR_antistasi_dialogs_generic_button_no_text,$STR_antistasi_dialogs_generic_button_yes_text};
+        default = 1;
```

Comment:
```suggestion
        default = 0;
```
Shouldn't be enabled by default imo